lib/ProductOpener/Display.pm

```diff
@@ -10599,6 +10614,24 @@ sub display_product_api ($request_ref) {
 			$response{jqm} .= $html;
 		}
 	}
+	elsif ((defined $product_ref->{product_type}) and ($product_ref->{product_type} ne $options{product_type})) {
```

Comment:
there again we have a complete duplication for product_multilingual.pl, that's a bit frustrating…